src/pages/search/project_search/ProjectFacetsFilter.tsx

```diff
@@ -34,13 +35,16 @@ export const ProjectFacetsFilter = ({ projectQuery }: ProjectFacetsFilterProps)
   const selectedParticipants = searchParams.get(ProjectSearchParameter.ParticipantFacet)?.split(',') ?? [];
   const selectedFundingSources = searchParams.get(ProjectSearchParameter.FundingSourceFacet)?.split(',') ?? [];
 
-  const addFacetFilter = (id: string) => {
-    const searchParameters = new URL(id).searchParams;
-    const newSearchParams = new URLSearchParams(
-      `${SearchParam.Type}=${currentSearchType}&${searchParameters.toString()}`
-    );
-    newSearchParams.set(SearchParam.Page, '1');
-    history.push({ search: newSearchParams.toString() });
+  const addFacetFilter = (param: string, key: string) => {
```

Comment:
Her kan vi være enda tydligere på typen til param. Ser jeg ikke har gjort det resultatsøk heller, men du kan sjå flisa i auga til bror din, men ikkje bjelka i di eiga auge osv.. :D 
```suggestion
  const addFacetFilter = (param: ProjectSearchParameter, key: string) => {
```